src/main/java/com/softdevsix/api/entities/PullRequest.java

```diff
@@ -0,0 +1,81 @@
+package com.softdevsix.api.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+/**
+ * Entity representing a pull request in the system.
+ * A pull request contains metadata such as its title and description,
+ * and it is associated with a specific project.
+ */
+@Setter
+@Getter
+@Entity
+public class PullRequest {
+
+    /**
+     * The unique identifier for the pull request.
+     */
+    @Id
+    @GeneratedValue
+    private UUID pullRequestId;
```

Comment:
Please review about SQL attributes naming convention please using the framework. Besides its generation using the annotation.